judgments/tests/test_detail.py

```diff
@@ -179,3 +179,42 @@ def test_non_canonical_uri_redirects(self, mock_judgment):
         assert isinstance(response, HttpResponseRedirect)
         assert response.status_code == 302
         assert response.url == "/test/1234/567"
+
```

Comment:
let's move these tests into a Different test class as they dont make sense to be grouped in TestDocumentURIRedirects.